pom.xml

```diff
@@ -8,8 +8,36 @@
     <artifactId>ariaddna-platform</artifactId>
     <packaging>pom</packaging>
     <version>1.0-SNAPSHOT</version>
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <configuration>
+                    <source>1.8</source>
+                    <target>1.8</target>
+                </configuration>
+            </plugin>
+        </plugins>
+    </build>
     <modules>
         <module>core</module>
         <module>rest-api-server</module>
+        <module>desktop-gui</module>
+        <module>desktop-core</module>
+        <module>common-utils</module>
     </modules>
+    <properties>
+        <spring.version>RELEASE</spring.version>
```

Comment:
Which framework version you are using here? 
I think we need to accurately describe the version, for ex.: 4.3.6.RELEASE